contracts/upgradeable_contracts/BasicBridge.sol

```diff
@@ -50,7 +50,12 @@ contract BasicBridge is
         return uintStorage[REQUIRED_BLOCK_CONFIRMATIONS];
     }
 
-    function claimTokens(address _token, address _to) public onlyIfUpgradeabilityOwner validAddress(_to) {
+    /**
+     * @dev Withdraws the erc20 tokens or native coins from this contract.
+     * @param _token address of the claimed token or address(0) for native coins.
+     * @param _to address of the tokens/coins receiver.
+     */
+    function claimTokens(address _token, address _to) public onlyIfUpgradeabilityOwner {
```

Comment:
can we check whether it is invoked only from the derived contracts? If so, should it be `internal`?

Reply:
Since this function contains only one line of code, I would prefer then to simply copy it there it is needed. In that case, we won't need internal functions at all. Also, it will be possible to mark those functions as `external` after such change.